src/marketing/products-viewed-compared.md

```diff
@@ -12,13 +12,19 @@ The Recently Viewed and Recently Compared blocks usually appear in the right sid
 
 1. Expand ![Expansion selector]({% link images/images/btn-expand.png %}) the **Recently Viewed/Compared Products** section and do the following:
 
-    ![Catalog configuration - recently viewed/compared products]({% link images/images/config-catalog-catalog-recently-viewed-compared-products.png %}){: .zoom}
+    ![Catalog configuration - recently viewed/compared products]({% link images/images/config-catalog-catalog-recently-viewed-compared-products1.png %}){: .zoom}
```

Comment:
Why are there two images in this PR? If you are just updating an existing image, delete the original and replace it.

Reply:
Hi @dobooth, thank you for reviewing!
I have deleted both images and replaced them with a new one.
Please, check it :)